environments/demo/backend_lb_config.yaml

```diff
@@ -192,6 +192,10 @@ gateways:
         component: judicial-api
       - product: rd
         component: judicial-data-load
+      - product: rd
+        component: location-ref-api
+      - product: rd
+        component: location-ref-data-load
```

Comment:
Data load won't have an ingress i guess? You don't need to add it here.

Reply:
@adusumillipraveen have removed data load